operators/math/math.cc

```diff
@@ -12,6 +13,8 @@ FxLoadCustomOpFactory LoadCustomOpClasses_Math =
                         CustomOpNegPos,
 #ifdef ENABLE_DLIB
                         CustomOpInverse,
+                        CustomOpStft,
```

Comment:
not sure if for ort-web or mobile we a single define ENABLE_DLIB is sufficient - I think we only want Stft